hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCommitLocks.java

```diff
@@ -226,6 +230,7 @@ public void testLockAcquisitionAfterFailedAndFoundLock() throws TException, Inte
 
     spyOps.doCommit(metadataV2, metadataV1);
 
+    verify(spyClient).showLocks(any()); // Make sure HiveLock's findLock method is called
```

Comment:
Could we please add `times(1)`, as it is a bit more specific what we expect:
```
verify(spyClient, times(1)).showLocks(any())
```

Reply:
done